### PR TITLE
Bug 1541618 - Add triage owner to edit components view columns.

### PR DESCRIPTION
**Pull request by kyoshino**
On the Edit Components page, replace the Watch User column with Triage Owner, and reorder the Active column.

## Bugzilla link

[Bug 1541618 - Add triage owner to edit components view columns.](https://bugzilla.mozilla.org/show_bug.cgi?id=1541618)

### CODE REVIEW

File: template/en/default/admin/components/list.html.tmpl [hunk 1]
```diff
@@ -105,6 +110,11 @@
        content => my_component.default_assignee.login
      }
   %]
+  [% overrides.triage_owner.name.${my_component.name} = {
+       override_content => 1
+       content => my_component.default_assignee.login
```
Comment (dklawren) [inline template/en/default/admin/components/list.html.tmpl]: my_component.triage_owner.login

Comment (kyoshino) [reply template/en/default/admin/components/list.html.tmpl]: Ouch. Thanks, fixing now :sweat_smile:

**Comment by kyoshino** [comment]
Somehow the Watch User column is not going away even though I have removed the hook and run `checksetup.pl`. Is the script not removing complied templates properly?

**Comment by dylanwh** [comment]
we shouldn't compile templates at all when MODE==development. Nuke template_cache/ for now.

**Comment by kyoshino** [comment]
Um, the column added by the hook still exists after ` rm -r template_cache/` 🤔

**Comment by kyoshino** [comment]
Figured out why: `vagrant rsync web` doesn’t remove deleted files because [`RSYNC_ARGS`](https://github.com/mozilla-bteam/bmo/blob/master/Vagrantfile#L19-L30) doesn’t have `--delete` option that can be found in the [default](https://www.vagrantup.com/docs/synced-folders/rsync.html#rsync__args). Can we add it?

**Pull request merged**